openfl-workspace/workspace/plan/defaults/tasks_connector.yaml

```diff
@@ -0,0 +1,4 @@
+start_client_adapter:
+  function : start_client_adapter
+  kwargs   :
+    local_server_port : 0 # local grpc server, 0 to dynamically allocate
```

Comment:
This may be a small item, but for a future PR I would like to see consistency between how gRPC ports are assigned. We use [the `auto` keyword today](https://github.com/securefederatedai/openfl/blob/develop/openfl/federated/plan/plan.py#L312) and have our own logic to calculate the port based on the hash of the plan. Because this is a local port that won't effect other distributed components, I don't think this is a big deal, but consistency in setting the port parameter would result in one less lookup for users.